.ci/scripts/build_ucc.sh

```diff
@@ -8,7 +8,8 @@ cd "${UCC_SRC_DIR}"
 mkdir -p "${UCC_SRC_DIR}/build"
 cd "${UCC_SRC_DIR}/build"
 "${UCC_SRC_DIR}/configure" --with-ucx="${UCX_INSTALL_DIR}" --with-cuda="${CUDA_HOME}" \
-    --prefix="${UCC_INSTALL_DIR}" --enable-gtest --with-mpi
+    --prefix="${UCC_INSTALL_DIR}" --enable-gtest --with-mpi \
+    --with-tls=cuda,nccl,self,ucp,mlx5,sharp,rccl
```

Comment:
i think no need to list tls here, tl mlx5 is built by default

Reply:
ok, removing it

Reply:
actually I think it is not built by default, check out the log:
`IBVERBS support: yes, MLX5DV support: yes`
and 
`configure:         TL modules:   < cuda nccl self ucp >`
so mentionning mlx5 seems needed here.

What to do with `sharp` and `rccl`

Reply:
ok, I'm asking because if you build it like this we won't be testing tl shm, which is not available in upstream

Reply:
Oh I see... What do you suggest then? For example, we could add `shm` here and a way to just skip it if not available. Or build mlx5 by default

Reply:
maybe use --with-tls=cuda,nccl,self,sharp,shm,ucp,mlx5

Reply:
done